Cargo.toml

```diff
@@ -24,7 +25,9 @@ byte-slice-cast = "1.2.2"
 clap = { version = "4.0", features = ["derive"] }
 criterion = { version = "0.4.0", features = ["html_reports"] }
 dashmap = "5.4.0"
+decompress = "0.1.0"
```

Comment:
do we still use the decompress crate? I thought that was the sync version that's now been replaced completely 🤔

Reply:
you're right, we don't!